### PR TITLE
Fixes #725

### LINKED ISSUE #725: System Install Fails Test
If pyne is installed as a system administrator (i.e. sudo setup.py install), one of the test cases fails when executed as a user, presumably because it does not have access to some file. This may not be of particular importance, just thought I'd mention it.

### PR DESCRIPTION
**Pull request by everythingfunctional**
- We'll make a local copy of nuc_data in case we don't have
  permissions to write to the original

### CODE REVIEW

File: tests/test_data_checksums.py [hunk 1]
```diff
@@ -35,9 +35,15 @@ def test_data_checksums():
 
 def test_internal_hashes():
     from pyne.dbgen import hashtools
-    hashtools.set_internal_hashes(pyne.nuc_data)
-    for item, val in hashtools.check_internal_hashes(pyne.nuc_data):
+    from shutil import copyfile
```
Comment (scopatz) [inline tests/test_data_checksums.py]: Import should go at the top of the file.

File: tests/test_data_checksums.py [hunk 2]
```diff
@@ -35,9 +35,15 @@ def test_data_checksums():
...
+    test_data = './test_nuc_data.h5'
```
Comment (scopatz) [inline tests/test_data_checksums.py]: The `./` here is not needed and platform dependent. Please remove.

File: tests/test_data_checksums.py [hunk 3]
```diff
@@ -35,9 +35,15 @@ def test_data_checksums():
...
+    # Create a copy so we don't try to modify a file we may not have
+    # permissions for
+    copyfile(pyne.nuc_data, test_data)
```
Comment (scopatz) [inline tests/test_data_checksums.py]: Please only copy the file if the user doesn't have write permission.  Right now it copies it all the time.  I think that you can use `os.stat()` to get the permissions.

File: tests/test_data_checksums.py [hunk 4]
```diff
@@ -35,9 +35,15 @@ def test_data_checksums():
...
+    hashtools.set_internal_hashes(test_data)
+    for item, val in hashtools.check_internal_hashes(test_data):
         assert_true(val)
+    os.remove(test_data)
```
Comment (scopatz) [inline tests/test_data_checksums.py]: This is only needed if the file was copied.

**Comment by scopatz** [comment]
Made a few comments, thanks for putting this in!

**Comment by everythingfunctional** [comment]
Made the changes you suggested.

**Comment by scopatz** [comment]
Really weird test failures.  Relaunching.

**Comment by scopatz** [comment]
The build errors are from uploading a new conda package.  I have put in a fix to #734  and pyne/pyne-ci#29.

**Comment by scopatz** [comment]
Relaunching after fix.

**Comment by scopatz** [comment]
Thanks a lot @bprichar! Sorry we ran into the CI issues here.

**Comment by everythingfunctional** [comment]
No problem. Glad I could help.

**Pull request merged**